Sources/WorkflowUIKit/UIKitPresenter.swift

```diff
@@ -61,6 +61,19 @@ open class UIKitPresenter: OrchestrationResponder {
         abandon(workflow, animated: true, onFinish: onFinish)
     }
 
+    public func complete(_ workflow: AnyWorkflow, passedArgs: AnyWorkflow.PassedArgs, onFinish: ((AnyWorkflow.PassedArgs) -> Void)?) {
+        // AnyWorkflow does not have a last(where:)
+        // swiftlint:disable:next last_where
+        let lastInstance = workflow.filter { $0.value.instance != nil }.last
```

Comment:
It wouldn't be hard to add.... Workflow has a last(where:) and you'd just have to call it, exactly like we did to make AnyWorkflow conform to `Sequence`

Reply:
Did the thing. Happy?

Reply:
Quite! 🎉